### PR TITLE
Update tests with new ctapipe-extra, remove hessio remnants

### PR DESCRIPTION
**Pull request by MaxNoe**
* Also gets rid of the duplicated environments by using sed to replace the python version before installing

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -37,8 +38,7 @@ dependencies:
   - wheel
   - xz
   - zlib
-  - pip
   - pip:
     - eventio~=1.0
-
-
+    - zstandard
```
Comment (watsonjj) [inline environment.yml:43]: What is this addition for?

Comment (MaxNoe) [reply environment.yml:43]: To read simtel files compressed using the zstd compression

Comment (MaxNoe) [reply environment.yml:43]: This was used already for prod4 and will i think be used for prod5 as well. Its much faster than gzip at a similar compression rate.

Comment (watsonjj) [reply environment.yml:43]: Isn't this already required by pyeventio then?

Comment (MaxNoe) [reply environment.yml:43]: It's an optional dependency of pyeventio, so we need to install ot explicitly

Comment (MaxNoe) [reply environment.yml:43]: Otherwise you will get a `You need zstandard to read this file` error

Comment (kosack) [inline environment.yml:43]: Move this out of pip install, it has a conda package and it's even the base anaconda distribution (not even conda-forge)

Comment (MaxNoe) [reply environment.yml:43]: Ah, nice. Sure

**Comment by MaxNoe** [comment]
@kosack @watsonjj pls review again

**Pull request merged**